clang/lib/Basic/Targets/LoongArch.h

```diff
@@ -132,6 +132,7 @@ class LLVM_LIBRARY_VISIBILITY LoongArch64TargetInfo
       : LoongArchTargetInfo(Triple, Opts) {
     LongWidth = LongAlign = PointerWidth = PointerAlign = 64;
     IntMaxType = Int64Type = SignedLong;
+    HasCheapUnalignedBitfieldAccess = true;
```

Comment:
Only set it to true when -mno-unaligned-access.

Reply:
Same to MIPS, while there is no -mno-unaligned-access for MIPS yet. I will add it support.

Reply:
> Only set it to true when -mno-unaligned-access.

In `LoongArchTargetInfo::handleTargetFeatures()`, we can handle the feature `-ual`.

Reply:
> > Only set it to true when -mno-unaligned-access.
> 
> In `LoongArchTargetInfo::handleTargetFeatures()`, we can handle the feature `-ual`.

Maybe it's true, while you may need it if you need to build a generic kernel image for both microarchitectures.

Reply:
you say set to true when -mno-unaligned-access, but I think you mean set to false in that case?

Reply:
> you say set to true when -mno-unaligned-access, but I think you mean set to false in that case?

Yes, I think it should be `false`. @wzssyqa

Reply:
fixed